.github/workflows/py_main.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Code Lint
         run: poetry run poe lint
       - name: Type check task with mypy
-        run: poetry run poe type-check --show-error-context --pretty
+        run: poetry run poe type-check -- -- --show-error-context --pretty
```

Comment:
for my own learning, what does this do? it's really odd looking

Reply:
First -- passes params to the poe 
Second -- passes args to the type-check script
Third -- exra arguments passed to the mapy

You don't need this if you use poe directly